src/web/event-notifications/event-notification-types/SlackNotificationSummary.jsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import CommonNotificationSummary from 'components/event-notifications/event-notification-types/CommonNotificationSummary';
+
+class SlackNotificationSummary extends React.Component {
+    static propTypes = {
+      type: PropTypes.string.isRequired,
+      notification: PropTypes.object,
+      definitionNotification: PropTypes.object.isRequired,
+    };
+
+    static defaultProps = {
+      notification: {},
+    };
+
+    render() {
+      const { notification } = this.props;
+
+      return (
+        <CommonNotificationSummary {...this.props}>
+          <>
+            <tr>
+              <td>Color</td>
+              <td>{notification.config.color}</td>
```

Comment:
Since we do not have `notifications.config` required or defaulted, we should verify they are set
```suggestion
              <td>{notification?.config?.color}</td>
```